app/presentation/RoomItem/Chip.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+import PropTypes from 'prop-types';
+
+import styles from './styles';
+import { themes } from '../../constants/colors';
+
+const Chip = React.memo(({ name, theme }) => (
+	<View style={{ backgroundColor: themes[theme].auxiliaryText, padding: 8 }}>
+		<Text
+			style={[
+				styles.title,
```

Comment:
It's not the same style as RoomItem.Title.
![image](https://user-images.githubusercontent.com/804994/118139158-78288d80-b3dd-11eb-8c1c-0c86b086b53f.png)